### ISSUE TITLE
very serious problem with k600 test set

### ISSUE THREAD

**Issue by tiancai-call** [cvdfoundation/kinetics-dataset#41]
After downloading, there are a lot of videos in the training set, and the actual number of correct videos is only more than 3,000

**Comment by tiancai-call** [comment]
Like -BC1iA8OOoQ_000006_000016.mp4, it should be in the training set, but it is in the test set you provided

**Comment by joaoluiscarreira** [comment]
Hi.

There is a large overlap between the kinetics700 training set and the
kinetics600 test set.
From the kinetics700 paper (https://arxiv.org/pdf/1907.06987v2.pdf):

*It is therefore largely safe to use models that have been trained on
Kinetics-600 to evaluate the Kinetics-700 test set (the activity-net
evaluation website explicitly ignores the predictions on those 3% clips
when evaluating on the test set). It is however not safe to train on
Kinetics-700 and then evaluate on Kinetics-600 – many of the
validation/test clips from Kinetics-600 are in the training set of
Kinetics-700 !*

Best,

Joao

**Comment by tiancai-call** [comment]
Thank you so much
